api/v1alpha3/zz_generated.deepcopy.go

```diff
@@ -448,6 +569,113 @@ func (in *VSphereMachineTemplateSpec) DeepCopy() *VSphereMachineTemplateSpec {
 	return out
 }
 
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *VSphereVM) DeepCopyInto(out *VSphereVM) {
+	*out = *in
+	out.TypeMeta = in.TypeMeta
+	in.ObjectMeta.DeepCopyInto(&out.ObjectMeta)
+	in.Spec.DeepCopyInto(&out.Spec)
+	in.Status.DeepCopyInto(&out.Status)
+}
+
+// DeepCopy is an autogenerated deepcopy function, copying the receiver, creating a new VSphereVM.
+func (in *VSphereVM) DeepCopy() *VSphereVM {
+	if in == nil {
+		return nil
+	}
+	out := new(VSphereVM)
+	in.DeepCopyInto(out)
+	return out
+}
+
+// DeepCopyObject is an autogenerated deepcopy function, copying the receiver, creating a new runtime.Object.
+func (in *VSphereVM) DeepCopyObject() runtime.Object {
+	if c := in.DeepCopy(); c != nil {
```

Comment:
Looking at line 596, it seems the nil check is not needed

edit: though this is generated code

Reply:
Hi @tedyu,

Yep, generated code. Resolving this comment. Thanks though!